recipes/torchx/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "torchx" %}
+{% set version = "0.7.0" %}
+{% set python_min = python_min|default("3.8") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pytorch/torchx/archive/v{{ version }}.tar.gz
+  sha256: 6a28e2a84dc9955856653fc6ce669fa36e522db82698426ace0ea54b0a02f9b8
+
+build:
+  entry_points:
+    - torchx = torchx.cli.main:main
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python {{ python_min }}
+    - setuptools
+  run:
+    - docker-py
+    - docstring_parser >=0.8.1
+    - filelock
+    - fsspec >=2023.10.0
+    - importlib-metadata
+    - pyre-extensions
+    - python >={{ python_min }}
+    - pyyaml
+    - tabulate
+    - urllib3 >=1.21.1,<1.27
+
+test:
+  imports:
+    - torchx
+  commands:
```

Comment:
We should probably test the cli here:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands